spec/dummy/app/resources/article_resource.rb

```diff
@@ -24,4 +24,8 @@ def id=(external_id)
   # Setting this attribute is an easy way to test that authorizations work even
   # when the model has validation errors
   attributes :blank_value
+
+  def self.creatable_fields(context)
+    super + [:id]
+  end
```

Comment:
Hmm I wonder if it has got to with @thibaudgg's PR #11 and the changes done to the dummy app `ArticleResource` in https://github.com/venuu/jsonapi-authorization/pull/11/files#diff-cc66704bb355f2769ceb1ed76d5f02b4

I'm not really sure how the `self.verify_key` and `id=` methods work, and why exactly they were necessary to have. @thibaudgg do you have any insight to this?